$OpenBSD: patch-i3status_conf,v 1.9 2012/10/09 16:02:07 jasper Exp $

Adjust config file to be reasonably useful on OpenBSD out of the box.

--- i3status.conf.orig	Wed Oct  3 13:44:44 2012
+++ i3status.conf	Tue Oct  9 19:44:34 2012
@@ -1,41 +1,43 @@
 general {
+        output_format = "i3bar"
         colors = true
         interval = 5
 }
 
 order += "ipv6"
 order += "disk /"
-order += "run_watch DHCP"
-order += "run_watch VPN"
-order += "wireless wlan0"
-order += "ethernet eth0"
+#order += "wireless iwn0"
+order += "ethernet em0"
 order += "battery 0"
 order += "load"
 order += "time"
+order += "cpu_temperature cpu0" 
+order += "cpu_temperature acpitz0"
+order += "volume master"
 
-wireless wlan0 {
-        format_up = "W: (%quality at %essid) %ip"
-        format_down = "W: down"
+cpu_temperature cpu0 {
+        format = "C: %degrees C"
+        path = "cpu0"
 }
 
-ethernet eth0 {
-        # if you use %speed, i3status requires root privileges
+cpu_temperature acpitz0 {
+        format = "TZ: %degrees C" 
+}
+
+#wireless iwn0 {
+#        format_up = "W: (%signal at %essid) %ip"
+#        format_down = "W: down"
+#}
+
+ethernet em0 {
         format_up = "E: %ip (%speed)"
         format_down = "E: down"
 }
 
 battery 0 {
-        format = "%status %percentage %remaining"
+        format = "%status %percentage \% %remaining min."
 }
 
-run_watch DHCP {
-        pidfile = "/var/run/dhclient*.pid"
-}
-
-run_watch VPN {
-        pidfile = "/var/run/vpnc/pid"
-}
-
 time {
         format = "%Y-%m-%d %H:%M:%S"
 }
@@ -46,4 +48,8 @@ load {
 
 disk "/" {
         format = "%free (or: %percentage_used used, %percentage_used_of_avail used of avail, %percentage_free free, %percentage_avail avail)"
+}
+
+volume master {
+	format = "vol: %volume"
 }
